package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "format-imports",
-  "version": "3.1.5",
+  "version": "4.0.0",
```

Comment:
Sorry for the confusion! I didn't mean to update this version. Please revert it and I'll update it if needed.